mirrorlist/mirrorlist_statistics.py

```diff
@@ -21,18 +21,18 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
 
-import sys, pylab, time, getopt
+import sys
+import pylab
+import time
+import getopt
 
 start = time.clock()
 
 logfile = None
 dest = None
 offset = 0
+embargoed_countries = ['CU', 'IR', 'KP', 'SD', 'SY']
```

Comment:
Hmm... Would make sense. Right now the script does not requite any MirrorManager infrastructure (configuration files, database access, imports). Which is nice as it can run anywhere as any user. If I start reading configuration files, which require special privileges, I could easily also connect to the database and read the list from there, which would make even more sense.

So either it will become less flexible by requiring certain hosts and users where and which it has to run, or it will stay the way it is now, but we have to maintain this list in the source code in addition to the list in the database. I have no preferences.